scripts/install_debs_cmake.sh

```diff
@@ -24,7 +24,7 @@ sudo apt-get install -y \
     g++ \
     git \
     google-mock \
-    libboost-all-dev \
+    libboost-iostreams-dev \
```

Comment:
This is really a minor point, but I'd rather not make changes to this file.  The simple reason is that right now, the only changes that we have to this repository compared to upstream are in the package.xml, and I'd like to keep it that way.  Anyone building from-source can probably afford to install the all-dev.

Reply:
fair enough, reverted in https://github.com/ros2/cartographer/pull/14/commits/2dc82c5c4ea53708e3c0a78942a23623a317d4f1

Reply:
Thanks!  Merging.